02/app.js

```diff
@@ -35,20 +41,24 @@ const checkIfPasswordsMatch = function (e, arr) {
   checkPassword(e, pass1, arr);
   const pass2 = e.target.elements.pass2.value;
   checkPassword2(e, pass2, arr);
+  const pass2Input = e.target.elements.pass2;
+  const label = pass2Input.parentNode.querySelector("label");
   if (pass1 !== pass2) {
-    const pass2Input = e.target.elements.pass2;
-    const label = pass2Input.parentNode.querySelector("label");
     arr.push(label);
     console.log(arr);
+  } else {
+    label.style.color = "black";
   }
 };
 
 const isRegChecked = function (e, arr) {
   const accept = e.target.elements["accept"];
+  const label = accept.parentNode.querySelector("label");
   if (!accept.checked) {
-    const label = accept.parentNode.querySelector("label");
     arr.push(label);
     console.log(arr);
+  } else {
+    label.style.color = "black";
```

Comment:
To jest ok, ale chyba łatwiej by było (mniej pisania) przed warunkiem `if(errors.length === 0)` wyszukać wszystkie `label` i dać im kolor `block` ;)